.gitignore

```diff
@@ -7,3 +7,4 @@ snap/.snapcraft
 .tox
 *.pyc
 .stestr
+.idea
```

Comment:
what is this?

Reply:
Pycharm IDE config file. Like this: https://github.com/openstack/nova/blob/master/.gitignore#L20